source/P0323R9.bs

```diff
@@ -762,10 +763,10 @@ the exception safety guarantee of `E`'s copy assignment.
 
 This operator is defined as deleted unless:
 
-  * `T` is *cv* `void` and `is_nothrow_move_constructible_v<E>` is `true` and `is_nothrow_move_assignable_v<E>` is `true`; or
+  * `T` is *cv* `void` and `is_move_constructible_v<E>` is `true` and `is_move_assignable_v<E>` is `true`; or
   * `T` is not *cv* `void` and `is_move_constructible_v<T>` is `true` and
-    `is_move_assignable_v<T>` is `true` and `is_nothrow_move_constructible_v<E>`
-    is `true` and `is_nothrow_move_assignable_v<E>` is `true`.
+    `is_move_assignable_v<T>` is `true` and `is_move_constructible_v<E>`
```

Comment:
The final "otherwise" clauses in the table can never be reached with the original constraints.